Homework 5-6

```diff
@@ -0,0 +1,98 @@
+HOMEWORK WEEK 5-6 
+(handout for students) 
+TASK 1 (Agile Techniques) 
+Question 1
+Complete definitions for Scrum related key terminology provided below. 
+SCRUM CEREMONIES 
+·  Product backlog refinement is a type of meeting held toward the end of the sprint to check that the backlog has been achieved before the next one
```

Comment:
This definition is not correct. 

PBR is  it is the act of refining items in the product backlog by adding detail, creating estimates and organising them. This is conducted by the product owner, in collaboration with the development team.  

This is an ongoing process in which the Product Owner and the Development Team collaborate on the details of Product Backlog items. During Product Backlog refinement, items are reviewed and revised.